recipes/default.rb

```diff
@@ -35,27 +35,34 @@
   notifies :reconfigure, 'chef_server_ingredient[supermarket]'
 end
 
-if node['supermarket_package']['package_source']
-  pkgname = ::File.basename(node['supermarket_package']['package_source'])
+if node['supermarket_omnibus']['package_url']
+  pkgname = ::File.basename(node['supermarket_omnibus']['package_url'])
   cache_path = ::File.join(Chef::Config[:file_cache_path], pkgname).gsub(/~/, '-')
 
   # recipe
   remote_file cache_path do
-    source node['supermarket_package']['package_source']
+    source node['supermarket_omnibus']['package_url']
     mode '0644'
   end
 end
 
+case node['platform_family']
+when 'debian'
+  node.default['apt-chef']['repo_name'] = node['supermarket_omnibus']['package_repo']
```

Comment:
This will only override the repo_name, but not the repo URL.   you'll need to override  name, url and key.